assessment/collectors/app_migration_analysis.go

```diff
@@ -317,11 +317,13 @@ func (m *MigrationSummarizer) AnalyzeFileTask(analyzeFileInput *AnalyzeFileInput
 		analyzeFileInput.filepath,
 		analyzeFileInput.methodChanges,
 		analyzeFileInput.content,
+		m.sourceSchema,
+		m.targetSchema,
 		analyzeFileInput.fileIndex)
 	return task.TaskResult[*AnalyzeFileResponse]{Result: analyzeFileResponse, Err: nil}
 }
 
-func (m *MigrationSummarizer) AnalyzeFile(ctx context.Context, projectPath, filepath, methodChanges, content string, fileIndex int) *AnalyzeFileResponse {
+func (m *MigrationSummarizer) AnalyzeFile(ctx context.Context, projectPath, filepath, methodChanges, content, sourceSchema, targetSchema string, fileIndex int) *AnalyzeFileResponse {
```

Comment:
why do you need to pass this as a parameter ?

it is already a part of the summarizer struct.

Reply:
reverted it, we can do refactoring in a separate PR.